packages/orchestration/src/utils/zoe.js

```diff
@@ -0,0 +1,56 @@
+import { Fail } from '@endo/errors';
+import { atomicTransfer } from '@agoric/zoe/src/contractSupport/index.js';
+
+/**
+ * @import {ChainAddress} from '../types.js';
+ * @import {OrchestrationAccount} from '../orchestration-api.js'
+ */
+
+export const makeZoeTools = (zone, { zcf, vowTools }) => {
+  /**
+   * Transfer the `give` a seat to a local account.
+   *
+   * @param {ZCFSeat} srcSeat
+   * @param {OrchestrationAccount<any>} localAccount
+   * @param {AmountKeywordRecord} give
+   * @returns {Promise<void>}
+   */
+  const localTransfer = vowTools.retriable(
```

Comment:
What's the best test to look at for this? `sendAnywhere.test.ts` should no longer have warnings about promises from `async-flow`?

Tangential: `swapExample.test.ts` has a test.skip we can hopefully restore soon.

Reply:
yes, the sendAnywhere test. I'm not aware of those warnings but if the test passes then this function works

Reply:
They look something like this on `master` and what we are trying to eliminate with #9449:

<details>
<summary>Warning for now: vow expected, not promise Promise { <pending> }</summary>

```bash
Warning for now: vow expected, not promise Promise { <pending> } (Error#1)
Error#1: where warning happened
  at tolerateHostPromiseToVow (.../async-flow/src/replay-membrane.js:157:9)
  at eval (.../common/object-map.js:45:40)
  at Array.map (<anonymous>)
  at objectMap (.../common/object-map.js:44:20)
  at tolerateHostPromiseToVow (.../async-flow/src/replay-membrane.js:168:8)
  at performCall (.../async-flow/src/replay-membrane.js:196:12)
  at guestCallsHost (.../async-flow/src/replay-membrane.js:256:9)
  at In "makeEmptySeatKit" method of (zcf) [as makeEmptySeatKit] (.../async-flow/src/replay-membrane.js:525:8)
  at withdrawFromSeat (.../zoe/src/contractSupport/zoeHelpers.js:226:52)
  at sendItFn (.../orchestration/src/examples/sendAnywhere.contract.js:71:24)
  
 Warning for now: vow expected, not promise Promise { <pending> } (Error#2)
Error#2: where warning happened
  at tolerateHostPromiseToVow (.../async-flow/src/replay-membrane.js:157:9)
  at eval (.../common/object-map.js:45:40)
  at Array.map (<anonymous>)
  at objectMap (.../common/object-map.js:44:20)
  at tolerateHostPromiseToVow (.../async-flow/src/replay-membrane.js:168:8)
  at performCall (.../async-flow/src/replay-membrane.js:196:12)
  at guestCallsHost (.../async-flow/src/replay-membrane.js:256:9)
  at In "makeEmptySeatKit" method of (zcf) [as makeEmptySeatKit] (.../async-flow/src/replay-membrane.js:525:8)
  at withdrawFromSeat (.../zoe/src/contractSupport/zoeHelpers.js:226:52)
  at sendItFn (.../orchestration/src/examples/sendAnywhere.contract.js:71:24)
  
  Warning for now: vow expected, not promise Promise { <pending> } (Error#3)
Error#3: where warning happened
  at tolerateHostPromiseToVow (.../async-flow/src/replay-membrane.js:157:9)
  at eval (.../common/object-map.js:45:40)
  at Array.map (<anonymous>)
  at objectMap (.../common/object-map.js:44:20)
  at tolerateHostPromiseToVow (.../async-flow/src/replay-membrane.js:168:8)
  at performCall (.../async-flow/src/replay-membrane.js:196:12)
  at guestCallsHost (.../async-flow/src/replay-membrane.js:256:9)
  at In "makeEmptySeatKit" method of (zcf) [as makeEmptySeatKit] (.../async-flow/src/replay-membrane.js:525:8)
  at withdrawFromSeat (.../zoe/src/contractSupport/zoeHelpers.js:226:52)
  at sendItFn (.../orchestration/src/examples/sendAnywhere.contract.js:71:24)
```
</details>

But are no longer present in this PR 🎉 . 


Hoping we can tighten the warning to an error, perhaps with the closing of #9449. We might also consider a parameter to asyncFlow tool to toggle the behavior while `E` support is being worked out.